src/views/errors/page-not-found.njk

```diff
@@ -0,0 +1,19 @@
+{% extends "shared/base.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% set pageTitleName = 'error.pageNotFound.title' | translate %}
```

Comment:
what purpose does setting this pageTitleName when we are setting a header below

Reply:
Currently the page title is output from nunjucks with a load of white space around the content - we can get rid of this by doing something like 
```
{% block pageTitle -%}
    {% if translate("pages."+hmpoPageKey+".serviceNameRequired") == 'true' %}
        {{ translate(hmpoPageTitle) }} – Prove your identity – GOV.UK
    {% else %}
        {{ translate(hmpoPageTitle) }} – GOV.UK
    {% endif %}
{%- endblock %}
```
it's the `{%-` and `-%}` that's doing the work trimming whitespace

Reply:
Also yes, there is some stuff that I put in about not showing the service name in the page `<title>` which can probably go